classes/npm.bbclass

```diff
@@ -18,6 +18,7 @@ NPM_INSTALL_FLAGS ?= ""
 NPM_REBUILD ?= "1"
 
 ISAR_CROSS_COMPILE = "0"
+SBUILD_FLAVOR = "${@ 'npm' if d.getVar('NPM_REBUILD') == '1' else '' }"
```

Comment:
We always need npm, also when not rebuilding. We need python3 and libnode108 on top when rebuilding, see below.

Reply:
Ok, but I want to avoid creating another schroot for the non-rebuild case. Shall we always run the build in the npm schroot?

Reply:
Just build a schroot that contains everything we need to installing and also rebuilding npm. That can be used for all npm packages, even if they don't have anything to rebuild.

Reply:
Fixed by making in unconditional.